README.md

```diff
@@ -14,16 +14,72 @@ Follow the prompts below and complete each question.  You may use any resource,
   +  `doctors` - use the first line of `data/doctors.csv` for the column names
   +  `patients` - use the first line of `data/patients.csv` for the column names
   +  `appointments` - which should have date, time, and comment columns (feel free to use the type `TEXT` for all three) as well as columns referencing both doctors and patients.
+
+CREATE TABLE doctors(
+  id SERIAL PRIMARY KEY,
+  surname VARCHAR,
+  given_name VARCHAR,
+  gender CHARACTER VARYING(1),
+  dob DATE,
```

Comment:
Given the format of the dates in the CSV file, I don't think that DATE will accept that format. STRING or TEXT would work.